simapp/app_config.go

```diff
@@ -137,7 +129,6 @@ var AppConfig = appconfig.Compose(&appv1alpha1.Config{
 					minttypes.ModuleName,
 					stakingtypes.BondedPoolName,
 					stakingtypes.NotBondedPoolName,
-					govtypes.ModuleName,
```

Comment:
why are we removing gov module from here ?

Reply:
This was a typo from a previous PR: https://github.com/cosmos/cosmos-sdk/blob/main/simapp/app_config.go#L140-L143. It should have only be commented and not blocked.